server/index.js

```diff
@@ -214,12 +215,14 @@ if (process.env.ENABLE_RESEARCHER_ACCESS && process.env.ENABLE_RESEARCHER_ACCESS
 
   // Endpoints for authenticated researchers to access data
   app.get('/server/research/data', [limiter, onlyAllowResearchers.bind(null, pool)], dataEndpoint.bind(null, pool));
+  app.get('/server/research/wav/(:id).wav', [limiter, onlyAllowResearchers.bind(null, pool)], audioEndpoint.bind(null, pool, config.s3));
 }
 
 // Serve any static files.
 // Route other requests return the React app, so it can handle routing.
 app.use(express.static(path.resolve(__dirname, '../client/build')));
 app.get('*', (request, response) => {
+  console.log('caught');
```

Comment:
leftover

Reply:
whoops. Again, should I reopen this branch to make this change or just do it in a patch on master?